src/migration.ts

```diff
@@ -82,7 +135,14 @@ async function getLastSuffix(
   }
 }
 
-export function getTimestamp(logger: Logger, filename: string): number {
+/**
+ * ! this function does not only resolve a timestamp numeric value. When prefix is numeric but neither 13 nor 17 digits, it returns the numeric prefix.
+ * ? should the function body be adjusted to do only what can be expected from it's name
+ */
+export function getTimestamp(
+  filename: string,
+  logger: Logger = console
```

Comment:
**todo:** check if getTimestamp is exported to external, and so if this is a breaking change or not

Reply:
it is not exported from the `index.js` file - however, since the package specifies wildcard exports, a consumer could do 

```js
import { getTimestamp } from 'node-pg-migrate/migration';
```

Reply:
agree 👍